.changeset/tasty-maps-jump.md

```diff
@@ -129,10 +129,8 @@ Status: replace emphasis tokens with 'emphasize' type
 Palette: token renaming emphasis to variants; color is the same where not stated
 
 ````diff
```

Comment:
even though this was done in that pr, should i just delete the notes about it here so its not confusing? @joshwooding